hivemq-edge/src/frontend/index.html

```diff
@@ -5,6 +5,7 @@
     <link rel="icon" type="image/svg+xml" href="/hivemq-edge.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>HiveMQ Edge</title>
+    <script type="module" src="/src/trackers.ts"></script>
```

Comment:
Have you checked that once the app is built and bundled, this location is still valid? I wouldn't have expected to see `/src` in there 🤔

Reply:
good point. It's not working

Reply:
In fact, it is working (forgot to turn the adblocker off on the prod domain!). But I'm refactoring the approach for a better readability